### DISCUSSION TITLE
Preset temperature problems

### DISCUSSION THREAD

**Discussion by danidv1969** [jmcollin78/versatile-thermostat-ui-card#39] [Ideas]
I am a beginner and therefore I may make mistakes.
Preset temperatures in ui-card works perfectly, but the new values ​​set by double clicking are not saved in versatile thermostat as new preset temperatures. As a result, when resetting the HA, these values ​​set in UI-card are lost. 
Thank you very much

![poza](https://github.com/user-attachments/assets/798aab37-ccaf-4c40-95ce-de5cec9d2df9)

In the photo is the UI-card and the 4 entities in the versatile thermostat

**Comment by mamema** [comment]
i cannot help you, because i didn't get that far. Even though configured central presets with the entities, when i double click on the "Activity detected" (motion sensors are activated) i get Failed to perform the action versatile_thermostat/set_preset_temperature. value must be one of ['boost', 'boost_ac', 'comfort', 'comfort_ac', 'eco', 'eco_ac', 'frost'] for dictionary value @ data['preset']

i don't know if this a ui card deb problem or a configuration issue.
